web/src/stream_edit.ts

```diff
@@ -804,6 +805,8 @@ export function initialize(): void {
             );
             if (sub && $subsection.attr("id") === "stream_permission_settings") {
                 stream_ui_updates.update_default_stream_and_stream_privacy_state($subsection);
+                const $edit_container = stream_settings_containers.get_edit_container(sub);
+                stream_ui_updates.update_can_subscribe_group_label($edit_container);
```

Comment:
The label is changed when changing stream privacy for an existing stream, even when "Save changes" button is not clicked. This behavior was also present for "Who can subscribe others to this channel" when we had the parens text for that setting.